packages/pigment-css-react/src/Grid.jsx

```diff
@@ -126,6 +126,12 @@ const GridComponent = styled('div')({
         marginLeft: 'var(--Grid-self-margin-left)',
       },
     },
+    ...['nowrap', 'wrap-reverse', 'wrap'].map((wrap) => ({
```

Comment:
```suggestion
    ...['nowrap', 'wrap-reverse'].map((wrap) => ({
```
To match the previous behavior.

Reply:
`wrap` already assigned when container is true. or remove the `flexWrap` from the container variant.

Reply:
can this prop be applied to non-container grid? If no, I can update the props definition to include `container: true`

Reply:
The previous implementation follows this ☝️ , I will update the code.

Reply:
> can this prop be applied to non-container grid

It should not. `flexWrap` only applied to flexbox container.